prov/efa/configure.m4

```diff
@@ -113,6 +115,44 @@ AC_DEFUN([FI_EFA_CONFIGURE],[
 	AC_SUBST(efa_CPPFLAGS)
 	AC_SUBST(efa_LDFLAGS)
 	AC_SUBST(efa_LIBS)
+
+	AC_ARG_WITH([cmocka],
+		[AS_HELP_STRING([--with-cmocka=DIR],
+				[Provide path to where the Cmocka runtime libraries
+				 are installed.])],
+		[efa_unit_test=yes], [efa_unit_test=no])
+
+	if [test x"$efa_unit_test" = x"yes"] && [test x"$with_cmocka" != x"yes"]; then
+		FI_CHECK_PACKAGE(cmocka,
+						 [cmocka.h],
+						 [cmocka],
+						 [_expect_any],
+						 [],
+						 [$with_cmocka],
+						 [$with_cmocka/lib64],
+						 [
+							AC_DEFINE([EFA_UNIT_TEST], [1], [EFA unit testing])
+							efatest_CPPFLAGS="$efatest_CPPFLAGS $efadv_CPPFLAGS -I$with_cmocka/include"
+							efa_CPPFLAGS="$efatest_CPPFLAGS $efadv_CPPFLAGS"
+							efa_LDFLAGS="-L$with_cmocka/lib64"
+							efatest_LDFLAGS="-L$with_cmocka/lib64"
+							efatest_LIBS=" -lcmocka -R$with_cmocka/lib64 -libverbs -static $efatest_LIBS"
+							efa_LIBS+=" -lcmocka -libverbs -static $efatest_LIBS"
+						 ],
+					 	 [
+							AC_MSG_WARN([Cannot compile EFA unit tests without cmocka.])
```

Comment:
should be AC_MSG_ERROR here because user explicitly asked to compile efa_test.

Reply:
done